compiler/src/iree/compiler/Dialect/Flow/Transforms/test/pipeline_tests.mlir

```diff
@@ -53,3 +53,50 @@ module {
 //       CHECK:     %[[T1:.+]] = flow.tensor.reshape %[[T0]] : tensor<833x833xf32> -> tensor<693889xf32>
 //       CHECK:     %[[T2:.+]] = flow.dispatch @[[EXECUTABLE1]]::@[[FUNC1]](%[[T1]])
 //       CHECK:     return %[[T2]]
+
+// -----
+
+#map = affine_map<(d0, d1, d2) -> (d0, d1, d2)>
+#map1 = affine_map<(d0, d1, d2) -> (d0, d1, 0)>
+#map2 = affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d3, d4)>
+#map3 = affine_map<(d0, d1, d2, d3, d4) -> (d2, d3, d4)>
+#map4 = affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d2)>
+module {
+  func.func @grouped_quantized_matmul(%arg0: tensor<4096x32x128xi4>, %arg1: tensor<1x1x32x128xf32>, %arg2: tensor<4096x32x1xf32>, %arg3: tensor<4096x32x1xf32>) -> tensor<1x1x4096xf32> {
+    %cst = arith.constant 0.000000e+00 : f32
+    %0 = tensor.empty() : tensor<1x1x4096xf32>
+    %1 = tensor.empty() : tensor<4096x32x128xf32>
+    %2 = linalg.fill ins(%cst : f32) outs(%0 : tensor<1x1x4096xf32>) -> tensor<1x1x4096xf32>
+    %3 = linalg.generic {indexing_maps = [#map, #map1, #map1, #map], iterator_types = ["parallel", "parallel", "parallel"]} ins(%arg0, %arg2, %arg3 : tensor<4096x32x128xi4>, tensor<4096x32x1xf32>, tensor<4096x32x1xf32>) outs(%1 : tensor<4096x32x128xf32>) {
+    ^bb0(%in: i4, %in_0: f32, %in_1: f32, %out: f32):
+      %5 = arith.extui %in : i4 to i32
+      %6 = arith.uitofp %5 : i32 to f32
+      %7 = arith.subf %6, %in_1 : f32
+      %8 = arith.mulf %7, %in_0 : f32
+      linalg.yield %8 : f32
+    } -> tensor<4096x32x128xf32>
+    %4 = linalg.generic {indexing_maps = [#map2, #map3, #map4], iterator_types = ["parallel", "parallel", "parallel", "reduction", "reduction"]} ins(%arg1, %3 : tensor<1x1x32x128xf32>, tensor<4096x32x128xf32>) outs(%2 : tensor<1x1x4096xf32>) {
+    ^bb0(%in: f32, %in_0: f32, %out: f32):
+      %5 = arith.mulf %in, %in_0 : f32
+      %6 = arith.addf %5, %out : f32
+      linalg.yield %6 : f32
+    } -> tensor<1x1x4096xf32>
+    return %4 : tensor<1x1x4096xf32>
+  }
+}
+
+//       CHECK:   flow.executable private @[[EXECUTABLE0:[a-zA-Z0-9_]+]]
+//       CHECK:   func.func @[[FUNC0:[a-zA-Z0-9_x]+]]
+//       CHECK:   linalg.generic
+//  CHECK-SAME:         ["parallel", "parallel", "parallel"]
```

Comment:
Please check that the result of the first generic is used in the second generic and that the second generic is an operand to `flow.return` of the executable.